pdc/apps/componentbranch/models.py

```diff
@@ -35,7 +34,6 @@ def export(self):
             'slas': [{'name': sla_to_branch.sla.name,
                       'eol': sla_to_branch.eol.strftime('%Y-%m-%d')}
                      for sla_to_branch in self.slas.all()],
-            'active': self.active,
```

Comment:
This should probably still be exported, even if it is not kept as a database member?

That way a query can quickly tell "is this thing active"?